components/sharing-dialog/i18n/en.pot

```diff
@@ -33,7 +33,7 @@ msgstr[1] ""
 msgid "These updated sharing settings will apply to {{count}} user or group."
 msgid_plural "These updated sharing settings will apply to {{count}} user or group."
 msgstr[0] "These updated sharing settings will apply to {{count}} user or group."
-msgstr[1] "These updated sharing settings will apply to {{count}} users and groups."
+msgstr[1] "These updated sharing settings will apply to {{count}} users or groups."
```

Comment:
I guess it can actually be both "and" and "or".

Reply:
Joe wanted "or" and far be it from me to argue with an Englishman :)